.github/workflows/release_nightlies.yml

```diff
@@ -125,40 +121,39 @@ jobs:
         env:
           RUSTFLAGS: -Ctarget-feature=+crt-static
 
-      - name: Build Mac / Ubuntu
-        if: matrix.os != 'windows-latest'
+      - name: Build macOS / Linux
+        if: matrix.os == 'macOS-latest' || matrix.os == 'ubuntu-latest'
         uses: actions-rs/cargo@v1
         with:
           command: build
           args: --release
 
       - name: Package common
         run: |
-          mkdir release
-          cp README.md release/README.md
-          cp LICENSE_APACHE release/LICENSE_APACHE
-          cp LICENSE_MIT release/LICENSE_MIT
+          mkdir package
+          cp README.md package/README.md
+          cp LICENSE_APACHE package/LICENSE_APACHE
+          cp LICENSE_MIT package/LICENSE_MIT
 
-      - name: Package windows
+      - name: Package Windows
         if: matrix.os == 'windows-latest'
         run: |
-          cp target/release/ruffle_desktop.exe release/ruffle.exe
-          7z a release.zip ./release/*
+          cp target/release/ruffle_desktop.exe package/ruffle.exe
+          7z a release.zip ./package/*
 
-      - name: Package linux/mac
+      - name: Package macOS / Linux
         if: matrix.os == 'macOS-latest' || matrix.os == 'ubuntu-latest'
         run: |
-          cp target/release/ruffle_desktop release/ruffle
-          cd release
-          tar -czvf ../release.tar.gz *
+          cp target/release/ruffle_desktop package/ruffle
+          tar -czvf release.tar.gz package/*
```

Comment:
I believe this adds the `package/` path into the zip file, which I'd like to avoid if possible.